src/app/design/styles/typography/ux-styles-typography.component.html

```diff
@@ -0,0 +1,454 @@
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class formats text to de-emphasize information that you generally want users to skip over, such as placeholder text.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <sky-code>&lt;div class=&quot;sky-deemphasized&quot;&gt;De-emphasized body copy&lt;/div&gt;</sky-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-italic.woff">Blackbaud Sans Italic</a> 15px #686c73</div>
+        </div>
+    </div>
+  </div>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-field-label">Field label</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class formats labels for read-only fields that do not accept input. It uses a muted color because the label that identifies the field is less important than the content.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <sky-code>&lt;div class=&quot;sky-field-label&quot;&gt;Field label&lt;/div&gt;</sky-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 15px #686c73</div>
+      </div>
+    </div>
+  </div>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-control-label">Form control label</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class formats labels for form input fields.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <sky-code>&lt;label class=&quot;sky-control-label&quot;&gt;Form control label&lt;/label&gt;</sky-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 15px #212327</div>
+        <div>margin-bottom: 5px</div>
+      </div>
+    </div>
+  </div>
+</div>
+
+ <sky-restricted-view>
+  <div *ngIf="themeSettings?.theme?.name === 'modern'" [skyTheme]="themeSettings">
+    <h2 class="sky-font-heading-2">Modern theme</h2>
+
+    <div class="sky-margin-stacked-content-section">
+    <h3 class="sky-font-heading-3">Body copy</h3>
+      <p>Body copy will generally make up the majority of text on a page.</p>
```

Comment:
We start the other sections with instruction of how to use the items; I think we should do the same for this. 

"Use body copy for the majority of the text on a page."

Reply:
updated